src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -79,6 +83,11 @@ class ItemRecognition extends Component {
           item.id === 'unknown' ? '/editsnack' : '/confirmitem'
         );
       } else {
+        if (
+          !this.state.showRetryMessage &&
+          (Date.now() - this.scanningStartTime) / 1000 > TIMEOUT_IN_SECONDS - 5
```

Comment:
Is it worth making this 5 into a constant? Just in case we decide to tweak it later, will make it easier to find.

Reply:
Think it is okay to leave as it's not being used elsewhere

Reply:
Just might be a bit awkward to locate in future if we want to alter it.